frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTable.tsx

```diff
@@ -137,60 +137,52 @@ const PoliciesTable = ({
         canAddOrDeletePolicy ? "" : "hide-selection-column"
       }`}
     >
-      {isLoading ? (
```

Comment:
Root issue: can't use a spinner to replace the table level or else it will re-render (and thus unfocus) the search input field